### ISSUE TITLE
NTR: Reduced O-mannosyl glycans on alpha-dystroglycan 

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4656]
**Preferred term label:**
Decreased O-mannosyl glycans on alpha-dystroglycan 

**Synonyms**
Reduced O-mannosyl glycans on alpha-dystroglycan 

**Definition (free text, please give PubMed ID)**
Hypoglycosylation of alpha-dystroglycan with O-mannosyl glycans. Alpha-dystroglycan is a functional target of O-mannosyl glycosylation and functional glycosylation of α-DG is essential in its interaction with the extracellular matrix. 
Source:  PMID:21930648

**Parent term (use hpo.jax.org/app)**
HP_0012358 'Abnormal protein O-linked glycosylation'

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.omim.org/entry/615042?search=615042&highlight=615042
(Note - I am reviewing the annotations for diseases listed in the GPI paper)

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-5208-3432

@pnrobinson I can add this if it looks okay

**Comment by pnrobinson** [comment]
@nicolevasilevsky this looks great!
